app/static/ringing_room.js

```diff
@@ -787,6 +811,489 @@ $(document).ready(function() {
+            
+            <br/>
+        </div>
+
+        <div id="wheatley_row_gen_box">
+            <!-- Wheatley row gen type toggle -->
+            <div class="btn-group btn-block btn-group-toggle">
+                <label class="btn btn-outline-primary"
+                       style="border-bottom-left-radius: 0;"
+                       :class="{active: row_gen_panel == 'method',
+                                disabled: row_gen_panel_disabled}">
+                    <input type="radio"
+                           name="row_gen_type"
+                           id="row_gen_type_method"
+                           value="method"
+                           v-model="row_gen_panel"
+                           />
+                    Method
+                </label>
+                <label class="btn btn-outline-primary"
+                       style="border-bottom-right-radius: 0;"
+                       :class="{active: row_gen_panel == 'composition',
+                                disabled: row_gen_panel_disabled}">
+                    <input type="radio"
+                           name="row_gen_type"
+                           id="row_gen_type_composition"
+                           value="composition"
+                           v-model="row_gen_panel"
+                           />
+                    Composition
+                </label>
+            </div>
+
+            <div id="wheatley_row_gen_box_inner"
+                 style="padding: 5px; 
+                        border: 1px solid white;
+                        border-top: none;
+                        border-bottom-right-radius: 0.25rem;
+                        border-bottom-left-radius: 0.25rem;
+                        margin-bottom: 1.2rem;">
+                <!-- Wheatley method tab -->
+                <div v-show="row_gen_panel == 'method'">
+                    <div>
+                        <form action="" @submit.prevent="on_method_box_enter">
+                            <input type="text"
+                                   id="wheatley_method_name_box"
+                                   class="form-control"
+                                   v-model="method_name"
+                                   placeholder="Start typing method name..."
+                                   :disabled="row_gen_panel_disabled"
+                                   autocomplete="off"
+                                   />
+                        </form>
+                        <div style="position: absolute;
+                                    display: inline-block;
+                                    z-index: 1;
+                                    width: 100%;
+                                    background-color: #666;
+                                    border-radius: 0.25rem;
+                                    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);">
+                            <a v-for="(suggestion, index) in autocomplete_options"
```

Comment:
I think it's the inline styles here that don't respect dark/light mode
![image](https://user-images.githubusercontent.com/10487726/93133395-0de50f00-f6cf-11ea-954c-75ae421effdb.png)

Reply:
Fixed `:)`